README.md

```diff
@@ -46,6 +46,19 @@ export APP_MAIL_PASSWORD=<app-mail-password>
 6. When you are done using the app, deactivate the virtual environment:
 `deactivate`
 
+### Run with docker
+1. Make sure you exported the following [environment variables](https://github.com/systers/mentorship-backend/wiki/Environment-Variables) to '.env' file
+
+2. Build docker image
+```
+docker build -t mentorship-backend:latest .
+```
+3. Deploy
+Docker container ports must be mapped to the host machine port using '--publish' so they're visible.
+```sh
+docker run --env "FLASK_APP=run.py" --publish 5000:5000 mentorship-backend:latest
```

Comment:
Do you need this flask app env variable here? isn't this solved with the new CMD on Docker file?